modules/installing-ocp-agent-verify.adoc

```diff
@@ -0,0 +1,90 @@
+// Module included in the following assemblies:
+//
+// * installing-with-agent/installing-with-agent.adoc
+
+:_content-type: PROCEDURE
+[id="installing-ocp-agent-verify_{context}"]
+= Tracking and verifying installation progress
```

Comment:
By reviewing this section I noticed that it lacks a lot of relevant information, that may be really important for the user (very likely an existing gap yet to be covered). @skopacz1 since they are not strictly related to the current task, do you think it would be possible to open a new ticket for this section?

Reply:
Sure thing, feel free to create an OSDOCS jira ticket, or I can create one and send it to you for editing/provision of context!

Reply:
Thanks! cc @betherly